.storybook/readme.md

```diff
@@ -0,0 +1,139 @@
+# Install:
```

Comment:
I think this file should be named storybook.md, and kept in root directory.

Reply:
Done